languages/rust/exercises/concept/enums/.docs/introduction.md

```diff
@@ -1 +1,3 @@
+## enums-basic
```

Comment:
@ErikSchierboom Are these headings presented as-is to the student, or are they parsed/prettified in any way? `enums-basic` is a good slug for a concept describing basic enums, but it's a pretty bad description to present to a human student.

Reply:
They will definitely be prettified. The most likely action is to replace them with the corresponding concept's `name` entry as defined in the `config.json` file.